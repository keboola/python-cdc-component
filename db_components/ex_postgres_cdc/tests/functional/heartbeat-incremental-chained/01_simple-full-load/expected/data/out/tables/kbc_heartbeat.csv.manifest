{"columns": ["id", "last_heartbeat", "KBC__OPERATION", "KBC__EVENT_TIMESTAMP_MS", "KBC__DELETED", "KBC__BATCH_EVENT_ORDER"], "enclosure": "\"", "delimiter": ",", "metadata": [], "column_metadata": {"id": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "INTEGER"}, {"key": "KBC.datatype.nullable", "value": false}, {"key": "KBC.datatype.length", "value": 10}], "last_heartbeat": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "TIMESTAMP"}, {"key": "KBC.datatype.nullable", "value": false}, {"key": "KBC.datatype.length", "value": 29}, {"key": "KBC.datatype.default", "value": "now()"}], "KBC__OPERATION": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": false}], "KBC__EVENT_TIMESTAMP_MS": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "TIMESTAMP"}, {"key": "KBC.datatype.nullable", "value": false}], "KBC__DELETED": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "BOOLEAN"}, {"key": "KBC.datatype.nullable", "value": false}], "KBC__BATCH_EVENT_ORDER": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "INTEGER"}, {"key": "KBC.datatype.nullable", "value": false}]}, "write_always": false}