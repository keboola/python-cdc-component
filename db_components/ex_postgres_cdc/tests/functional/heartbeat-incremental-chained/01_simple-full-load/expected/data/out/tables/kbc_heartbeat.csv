1,r,false,0
